docs/fundamentals.md

```diff
@@ -1,22 +1,24 @@
+The fundamentals describe how LeakCanary works and how to use it to detect and fix memory leaks. This documentation is designed to help developers at all levels, so please don't hesitate to report any confusing section.
```

Comment:
`at all levels` or `of all levels`? Does it refer to developer's level or to the levels of detecting a leak?

Reply:
The developers level. Not sure which one of these construct is right, I hesitated